src/main/java/com/google/codeu/data/Message.java

```diff
@@ -41,6 +42,18 @@ public Message(UUID id, String user, String text, long timestamp) {
     this.timestamp = timestamp;
   }
 
+  public Message(String user, String text, String recipient) {
+    this(UUID.randomUUID(), user, text, System.currentTimeMillis(), recipient);
```

Comment:
This constructor doesn't seem to be used. Is it needed?

Reply:
I think the guide said that we were just adding in the constructor but not using it yet, to prep all the Java files for working with the recipient parameter.

Reply:
I'm also new to the familiarization project so correct me if I'm wrong, but I don't see where the guide says to add the constructors. To me, it seems like it asks to add the recipient parameter to the existent constructors.

Reply:
https://sites.google.com/codeustudents.com/spring-2019/weeks-1-2-familiarization-projects/direct-messages?authuser=1#h.p_wF3N6-wM4b6g

Reply:
I read over the guide again and you're right - I had thought it meant to add another version of the constructor but I've updated it so that I just added the recipient parameter to the existing constructors.